docs/overview.md

```diff
@@ -40,7 +40,9 @@ IntelliCode provides AI-assisted IntelliSense in both Visual Studio and Visual S
 The completion list suggests the most likely correct API for a developer to use rather than presenting a simple alphabetical list of members. To provide this dynamic list, IntelliCode uses the developer's current code context as well as patterns based on thousands of highly rated, open-source projects on GitHub. The results form a model that predicts the most likely and most relevant API calls.
```

Comment:
"correct" doesn't sound right because every item in the IntelliSense menu is correct. Perhaps, "most likely to be used API"

Reply:
new language: AI-assisted IntelliSense predicts and presents the most likely to be used API at the top of the IntelliSense window rather than just presenting an alphabetical list of members.